From d2705840d43132bddc698e78d272f0014603ad7a Mon Sep 17 00:00:00 2001
From: Bin Yang <yangbin@rock-chips.com>
Date: Wed, 19 Oct 2016 15:39:45 +0800
Subject: [PATCH] arm64: dts: rockchip: add sensor lsm330 node for rk3399-mid

Add the sensor node "power-off-in-suspend" to support sensor poweroff in
system suspend.

Change-Id: Ic0dc660f0211b7dd18ba8fec6d54aba5b1dfc301
Signed-off-by: Bin Yang <yangbin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
index 4d54b95c1da0..70f5273b2865 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
@@ -864,6 +864,7 @@
 		type = <SENSOR_TYPE_ACCEL>;
 		irq_enable = <1>;
 		poll_delay_ms = <30>;
+		power-off-in-suspend = <1>;
 		layout = <4>;
 	};
 
@@ -876,6 +877,7 @@
 		irq-gpio = <&gpio1 20 IRQ_TYPE_EDGE_RISING>;
 		type = <SENSOR_TYPE_GYROSCOPE>;
 		irq_enable = <0>;
+		power-off-in-suspend = <1>;
 		poll_delay_ms = <30>;
 	};
 
-- 
2.35.3

